scripts/Drive.ps1

```diff
@@ -68,12 +68,18 @@ function Get-SwordFishDrive{
                            @{@odata.id=/redfish/v1/StorageServices/S1/Drives/1.4}...}
 .LINK
     https://redfish.dmtf.org/schemas/v1/Drive.v1_11_0.json
+    The Drives and Drives Collections will existing in the following Locations    
+        /redfish/v1/Drives/{DriveId}
+        /redfish/v1/Chassis/{ChassisId}/Drives/{DriveId}
+        /redfish/v1/Storage/{StorageId}/Drives/{DriveId}
```

Comment:
Is this supposed to be a reference collection? If so, OK.   But drives only have two locations that are acceptable moving forward.  Most should be in /redfish/v1/Chassis/../Drives

Alternate is /redfish/v1/StorageServices/.../Drives

Reply:
Yes, indeed this is for a collection. The command can be directed to return collections, or the drives themselves, depending on the arguments used.